src/modules/SX126x/SX126x.h

```diff
@@ -499,6 +499,36 @@ class SX126x: public PhysicalLayer {
     */
     int16_t startReceive(uint32_t timeout = SX126X_RX_TIMEOUT_INF);
 
+    /*!
+      \brief Interrupt-driven receive method where the device mostly sleeps and periodically wakes to listen.
+
+      \param rxPeriod_us The duration the receiver will be in Rx mode, in microseconds.
+
+      \param sleepPeriod_us The duration the receiver will not be in Rx mode, in microseconds.
+
+      \returns \ref status_codes
+
+      Note that this function assumes the unit will take 500us to change state. See datasheet [SECTION].
```

Comment:
Please include datasheet section and version when quoting datasheets.

Reply:
ah yes. That was the intention, hence the placeholder. Done.